data/campaigns/Eastern_Invasion/units/Egg_Sac.cfg

```diff
@@ -0,0 +1,40 @@
+#textdomain wesnoth-ei
+[unit_type]
+    id=Egg Sac
+    name= _ "Egg Sac"
+    race=monster
+    image="units/ant/egg-sac.png"
+    hitpoints=28
+    movement_type=none
+    movement=0
+    experience=25
+    level=0
+    alignment=neutral
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=31
+    usage=fighter
+    description= _ "Each of these enormous egg sacs contains up to a dozen ant pupae, sheltering and nourishing them while they develop into worker drones. When the burrow is threatened, an ant queen can release pheromones that cause each sac to rapidly develop into a single large warrior ant."
```

Comment:
This description is sending mixed messages. If they're egg sacs then they should contain eggs, which hatch into larvae, which at some point form a cocoon and turn into pupae. So, are they egg sacs? Or are they… I don't know if this is even a real term… pupal sacs? It doesn't make sense for them to be both.